From 3ec35f182ccb9d47eef02a44e7c708c0ea3b330a Mon Sep 17 00:00:00 2001
From: Varun Kumar Reddy Yaparla <vyaparla@habana.ai>
Date: Wed, 19 Jan 2022 13:24:13 +0200
Subject: [PATCH] Revert "[SW-70899] gaudi2: arc: reset CPU_PLL"

This reverts commit 3bdcd9b24bc738fd5cc717473635bbb3f952a19f.

Maxim is concerned about some un-known bugs getting discovered when CPU
PLL is reset as part of PRSTN reset. Even if PSOC is reset there are
some blocks/parts still active and CPU PLL reset may cause issues

Will add other patch to revert RR/protection bits programming done to
CPU PLL block when fake security is disabled

Change-Id: I714663f29e6d17a54433fe351be97e20ad336b81
---

diff --git a/arch/gaudi2/arc/common.c b/arch/gaudi2/arc/common.c
index c5bb6c6..e91149a 100644
--- a/arch/gaudi2/arc/common.c
+++ b/arch/gaudi2/arc/common.c
@@ -65,13 +65,6 @@
 	regmap_writel(0x1, mmPSOC_RESET_CONF_PSOC_PRSTN_RST_CFG);
 	regmap_writel(0x3, mmPSOC_RESET_CONF_ARC_PRSTN_RST_CFG);
 
-	/*
-	 * Reset value of PLL_L_PRSTN_RST_CFG is 0xFFFFFFFD, BIT(1) which corresponds to CPU_PLL
-	 * is not getting reset by default. Now ARCs are getting reset as part of Warm/PRSTn
-	 * reset, CPU_PLL should also need to be resetted
-	 */
-	regmap_writel(0xFFFFFFFF, mmPSOC_RESET_CONF_PLL_L_PRSTN_RST_CFG);
-
 	/* Clear masking of PRSTN from SW reset */
 	regmap_writel(0x0, mmPSOC_GLOBAL_CONF_PRSTN_MASK);
 
